### ISSUE TITLE
country code on iOS 16.4.x is always '--'

### ISSUE THREAD

**Issue by efstathiosntonas** [dev-family/react-native-device-country#7]
Hi, I'm using version `1.0.3` and iOS 16.4, the country code being returned is always '--' when using `DeviceCountry.getCountryCode()`.

result: 
`{"code": "--", "type": "telephony"}`

On previous iOS versions this was working fine. 

I'm using react-native `0.71.7`

**Comment by HERYORDEJY** [comment]
@efstathiosntonas  I found a workaround
```javascript
DeviceCountry.getCountryCode(
  Platform.OS === 'ios' ? TYPE_CONFIGURATION : TYPE_TELEPHONY,
)
  .then(result => {
      //
    }
  )
```

**Comment by efstathiosntonas** [comment]
Thanks @HERYORDEJY-DEV, will try it out!

**Comment by mshekspirtt** [comment]
From IOS 16.0 [CTCarrier was deprecated](https://developer.apple.com/documentation/coretelephony/cttelephonynetworkinfo/3024511-servicesubscribercellularprovide/). 
Released a new version 1.0.4, but on IOS it is now possible to get only the country code from the phone settings and passing types are not supported.
